azure_encryption_helper/encryptor.py

```diff
@@ -0,0 +1,47 @@
+from .aescipher import AESCipher
+from .aeskeywrapper import AESKeyWrapper
+
+import os
+
+class Encryptor:
+
+    def __init__(self, key=None, iv_length=16):
+        if key is None:
+            key = Encryptor.generate_aes_key()
+        self.key = key
+        self.iv_length = iv_length
+        self._update_cipher()
```

Comment:
I don't we should `_update_cipher()` if the Key is None.

Reply:
I think this still works, if a key isn't provided, one will be generated and used for the instance of cipher in this `Encryptor` object

Reply:
Right, if you don't provide the key, Encryptor generates one for you